ext/dom/dom_properties.h

```diff
@@ -19,127 +19,125 @@
-int dom_document_encoding_write(dom_object *obj, zval *newval);
-int dom_document_standalone_read(dom_object *obj, zval *retval);
-int dom_document_standalone_write(dom_object *obj, zval *newval);
-int dom_document_version_read(dom_object *obj, zval *retval);
-int dom_document_version_write(dom_object *obj, zval *newval);
-int dom_document_strict_error_checking_read(dom_object *obj, zval *retval);
-int dom_document_strict_error_checking_write(dom_object *obj, zval *newval);
-int dom_document_document_uri_read(dom_object *obj, zval *retval);
-int dom_document_document_uri_write(dom_object *obj, zval *newval);
-int dom_document_config_read(dom_object *obj, zval *retval);
-int dom_document_format_output_read(dom_object *obj, zval *retval);
-int dom_document_format_output_write(dom_object *obj, zval *newval);
-int	dom_document_validate_on_parse_read(dom_object *obj, zval *retval);
-int dom_document_validate_on_parse_write(dom_object *obj, zval *newval);
-int dom_document_resolve_externals_read(dom_object *obj, zval *retval);
-int dom_document_resolve_externals_write(dom_object *obj, zval *newval);
-int dom_document_preserve_whitespace_read(dom_object *obj, zval *retval);
-int dom_document_preserve_whitespace_write(dom_object *obj, zval *newval);
-int dom_document_recover_read(dom_object *obj, zval *retval);
-int dom_document_recover_write(dom_object *obj, zval *newval);
-int dom_document_substitue_entities_read(dom_object *obj, zval *retval);
-int dom_document_substitue_entities_write(dom_object *obj, zval *newval);
+zend_result dom_document_doctype_read(dom_object *obj, zval *retval);
+zend_result dom_document_implementation_read(dom_object *obj, zval *retval);
+zend_result dom_document_document_element_read(dom_object *obj, zval *retval);
+zend_result dom_document_actual_encoding_read(dom_object *obj, zval *retval);
+zend_result dom_document_actual_encoding_write(dom_object *obj, zval *newval);
+zend_result dom_document_encoding_read(dom_object *obj, zval *retval);
+zend_result dom_document_encoding_write(dom_object *obj, zval *newval);
+zend_result dom_document_standalone_read(dom_object *obj, zval *retval);
+zend_result dom_document_standalone_write(dom_object *obj, zval *newval);
+zend_result dom_document_version_read(dom_object *obj, zval *retval);
+zend_result dom_document_version_write(dom_object *obj, zval *newval);
+zend_result dom_document_strict_error_checking_read(dom_object *obj, zval *retval);
+zend_result dom_document_strict_error_checking_write(dom_object *obj, zval *newval);
+zend_result dom_document_document_uri_read(dom_object *obj, zval *retval);
+zend_result dom_document_document_uri_write(dom_object *obj, zval *newval);
+zend_result dom_document_config_read(dom_object *obj, zval *retval);
+zend_result dom_document_format_output_read(dom_object *obj, zval *retval);
+zend_result dom_document_format_output_write(dom_object *obj, zval *newval);
+zend_result dom_document_validate_on_parse_read(dom_object *obj, zval *retval);
```

Comment:
Was that a sneaky tab between the int and the function name before :thinking:

Reply:
yep